Features/ContentActions/CopyContent.feature

```diff
@@ -15,7 +15,7 @@ Feature: Copy content
         And a "News Flash" article exists
         And I am on "News Flash" full view
         When I click on the action bar button "Copy"
-        And I select the "eZ Platform/Destination" folder in the Universal Discovery Widget
+        And I select the "eZ Platform/Destination" folder in the Finder method of Universal Discovery Widget
```

Comment:
Will it break if we just replace the existing sentence with this new implementation ? We don't really care how the method is selected, do we ? We just want to select something.